tests/integration/test_charm.py

```diff
@@ -124,10 +125,47 @@ async def test_integration_with_s3_integrator(
     )
 
     # Assert that both kyuubi-k8s and s3-integrator charms are in active state
-    assert ops_test.model.applications[APP_NAME].status == "active"
+    assert ops_test.model.applications[APP_NAME].status == "blocked"
     assert ops_test.model.applications[charm_versions.s3.application_name].status == "active"
 
 
+@pytest.mark.abort_on_fail
+async def test_integration_with_integration_hub(
+    ops_test: OpsTest, charm_versions, s3_bucket_and_creds
```

Comment:
chore: remove `s3_bucket_and_creds` which is not referenced anywhere

Reply:
I believe it is a fixture in conftest.py

Reply:
Yes, but I believe this fixture is not required in this particular function?